backstage.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "1.22.1"
+  "version": "1.23.0"
```

Comment:
Let's bump this to 1.23.1 like https://github.com/spreadshirt/backstage-plugin-s3/pull/83.

Reply:
Done. I just skipped it because the affected package was not used here